lib/rollup.config.js

```diff
@@ -1,36 +1,82 @@
 import babel from 'rollup-plugin-babel';
 import resolve from 'rollup-plugin-node-resolve';
 import commonjs from 'rollup-plugin-commonjs';
+import { uglify } from 'rollup-plugin-uglify';
 import { sizeSnapshot } from 'rollup-plugin-size-snapshot';
 import pkg from './package.json';
 
 // treat as externals not relative, not absolute and not reserved rollup paths
-const external = id =>
+const cjsExternal = id =>
   !id.startsWith('\0') && !id.startsWith('.') && !id.startsWith('/');
 
-export default {
-  input: './src/index.js',
-  output: [
-    { file: pkg.main, format: 'cjs', sourcemap: true },
-    { file: pkg.module, format: 'esm', sourcemap: true },
-  ],
-  external,
-  plugins: [
-    resolve({
-      extensions: ['.js', '.jsx'],
-    }),
-    babel({
-      exclude: 'node_modules/**',
-      plugins: ['external-helpers'],
-    }),
-    commonjs({
-      include: [
-        'node_modules/**',
-      ],
-      namedExports: {
-        'node_modules/moment-range/dist/moment-range.js': ['extendMoment'],
-      },
-    }),
-    sizeSnapshot()
+const input = './src/index.js';
+const globals = {
+  react: 'React',
+  'react-dom': 'ReactDOM',
+  'prop-types': 'PropTypes',
+};
+
+const babelOptions = {
```

Comment:
This was a function for a future reason.

Reply:
👍